src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstNumber) => {
+    counter++;
+
+    const isError = counter > 3 && counter % 2 === 0;
+
+    return (secondNuNumber) => {
+      if (isError) {
+        return 'Bzzz... Error!';
+      }
+
+      return firstNumber + secondNuNumber;
+    };
```

Comment:
You can make it even better with ternary
```suggestion
    return (secondNumber) => (
      isError
       ? 'Bzzz... Error!'
       : firstNumber + secondNumber
    );
```